### PR TITLE
Add 4 board themes

### PR DESCRIPTION
**Pull request by Khislatjon**
Added 4 themes: Classic, Wood, Light and Glass.

https://github.com/nicklockwood/Chess/assets/58033343/3396db26-bb32-4c6e-b40b-c902a4921a08

### CODE REVIEW

File: Chess/BoardView.swift [hunk 1]
```diff
@@ -171,6 +175,20 @@ class BoardView: UIView {
         updatePieces()
         updateMoveIndicators()
     }
+    
+    private func updateTheme() {
```
Comment (Khislatjon) [inline Chess/BoardView.swift:179]: Please let me know if there is an optimization for this function. I don't like inserting new view to the bottom of a view hierarchy. But couldn't find a better way.

**Comment by nicklockwood** [comment]
These themes look great! But any reason why you chose to use images rather than just color for the themes? It seems like colors would be simpler to work with and simplify the process of adding more themes in future (or even letting the user create their own)

**Comment by Khislatjon** [comment]
There were already 2 images with colors. I thought to add more. But you are right, I will replace them with colors. Thanks.

**Comment by Khislatjon** [comment]
@nicklockwood Updated PR.

**Pull request merged**